threesixty/test_settings.py

```diff
@@ -0,0 +1,10 @@
+from threesixty.settings import *
+
+DATABASES = {
```

Comment:
PostgreSQL is available on travis-ci, please use it ;)
You just need to add the addon and define your own DATABASE_URL. But it's all in the travis-ci documentation.